src/components/20-molecules/policy-features/policy-features-item/index.scss

```diff
@@ -18,6 +18,22 @@
 
   &__icon {
     max-width: 100%;
+    color: $color-prim-blue-dark-indigo; // default background color for svgs
+
+    // because of fill=currentColor we need to set color to the backgroundColor of the variant
+    &__style {
```

Comment:
`style-axa-blue` is not an element, it should be a modifier.

`__icon--axa-blue` for instance. I would leave out the "style" in the property name.

Reply:
Oh, the "__" and "--" have a deeper meaning? Will fix that...